### DISCUSSION TITLE
Connecting with a non-official version of Minecraft

### DISCUSSION THREAD

**Discussion by TimDrms** [ORelio/Minecraft-Console-Client#1983] [Q&A] [answered]
Hey, I just wonder if we can use MCC without having an official Minecraft account.
My goal is to connect to a a Minecraft server that is open to non-official version of Minecraft with an alt in order to afk with it.

:)

**Comment by ReinforceZwei** [comment, marked as answer]
Input `-` for account password to enter offline mode.
